rust/noosphere-cli/src/native/mod.rs

```diff
@@ -96,7 +96,11 @@ pub enum OrbCommand {
 
     /// Show details about files in the sphere directory that have changed since
     /// the last time the sphere was saved
-    Status,
+    Status {
+        /// Output a single attribute of the status
+        #[clap(short, long, default_value="")]
+        attr: String,
```

Comment:
If you make this field `id: bool` then it will automatically give you a `true` or `false` depending on if `--id` is used.

Then, the `status` command function can accept an `only_id` boolean.

Reply:
Is there harm in outputting all the status info, rather than needing to specify a single attribute (IIUC)?

Reply:
There is no harm, it's more about scope. The original change was pretty simple, but `orb status` is manually formatted and producing JSON output would be a significant increase in scope for this change. Ideally, we would have machine-friendly and human-friendly outputs for most commands.

Reply:
I'm not opposed to a json output.. it's just beyond my rust skills at the moment.

Reply:
For sure, `--json` or whatever is a bigger task for the future